cmd/check.go

```diff
@@ -143,6 +143,14 @@ var checkCmd = &cobra.Command{
 				minVersion: "2.17.1",
 				docsURL:    "https://git-scm.com/book/en/v2/Getting-Started-Installing-Git",
 			},
+			{
```

Comment:
Can you also bump the terraform to 0.13 please, the EKS stack's terraforms are now using some 0.13 features